gravitee-apim-console-webui/src/management/api/runtime-logs-v4/runtime-logs-settings/api-runtime-logs-settings.component.html

```diff
@@ -0,0 +1,148 @@
+          <mat-form-field class="settings__form-control__form-field">
+            <mat-label>Message sampling</mat-label>
+            <input matInput type="number" formControlName="samplingValue" />
+            <mat-error *ngIf="form.get('samplingValue').hasError('required')"> The sampling value is required. </mat-error>
+            <mat-error *ngIf="form.get('samplingValue').hasError('min')"> The sampling value should be greater than 0.01. </mat-error>
+            <mat-error *ngIf="form.get('samplingValue').hasError('max')"> The sampling value should be lower than 0.5. </mat-error>
+          </mat-form-field>
+        </ng-container>
+
+        <ng-container *ngIf="samplingType === 'COUNT'">
+          <p>Samples one message for every number of specified messages.</p>
+          <p>The value should be greater than 10.</p>
+          <mat-form-field class="settings__form-control__form-field">
+            <mat-label>Message sampling</mat-label>
+            <input matInput type="number" formControlName="samplingValue" />
+            <mat-error *ngIf="form.get('samplingValue').hasError('required')"> The sampling value is required. </mat-error>
+            <mat-error *ngIf="form.get('samplingValue').hasError('min')"> The sampling value should be greater than 0. </mat-error>
+          </mat-form-field>
+        </ng-container>
+
+        <ng-container *ngIf="samplingType === 'TEMPORAL'">
+          <p>Samples messages based on time duration.</p>
+          <p>The value should conform to ISO-8601 duration format, e.g. PT10S.</p>
+          <mat-form-field class="settings__form-control__form-field">
+            <mat-label>Message sampling</mat-label>
+            <input matInput formControlName="samplingValue" />
+            <mat-error *ngIf="form.get('samplingValue').hasError('required')"> The sampling value is required. </mat-error>
+          </mat-form-field>
+        </ng-container>
+      </div>
+
+      <div class="settings__form-control">
+        <div class="settings__form-control__title">Display conditions</div>
+        <div class="settings__form-control__subtitle">Filter the data that matter to you..</div>
+
+        <mat-form-field>
+          <mat-label>Request phase condition</mat-label>
+          <input matInput formControlName="requestCondition" />
+          <mat-hint> Support EL (e.g: &#123; #request.headers['Content-Type'][0] === 'application/json' &#125;)</mat-hint>
+        </mat-form-field>
+
+        <mat-form-field class="settings__form-control__form-field">
+          <mat-label>Message condition</mat-label>
+          <input matInput formControlName="messageCondition" />
+          <mat-hint> Support EL (e.g: &#123; #message.headers['Content-Type'][0] === 'application/json' &#125;)</mat-hint>
+        </mat-form-field>
+      </div>
+    </div>
+
+    <div>
+      <button
+        class="settings__save"
+        [disabled]="form.invalid"
+        *gioPermission="{ anyOf: ['api-definition-u'] }"
```

Comment:
**thought**:
It reminds me that we should dig into the difference between `api-definition` & `api-gateway-definition` permissions. Because I tend to think that here it should be "gateway-definition" but I'm not sure.